modules/user/client-react/containers/UserAdd.jsx

```diff
@@ -56,16 +56,7 @@ export default compose(
             variables: { input }
           });
 
-          if (addUser.errors) {
-            return { errors: addUser.errors };
-          }
-
-          if (history) {
-            return history.push('/users/');
-          }
-          if (navigation) {
-            return navigation.goBack();
-          }
+          return addUser;
         } catch (e) {
           console.log(e.graphQLErrors);
```

Comment:
Why do we leave silly error handling code like this? Maybe now we can remove try catch block?

Reply:
fix